### DISCUSSION TITLE
weewx-wdc Gauges

### DISCUSSION THREAD

**Discussion by Ecloud44** [Daveiano/weewx-wdc#288] [Q&A] [answered]
Hi , 
i just installed weewx-dc on a Debian 12.7 
PHP 8.3.14 (cli) (built: Nov 25 2024 18:23:27) (NTS)
Server version: Apache/2.4.62 (Debian)
Weewx 5.1 fresh install 

I have enable gauge in skin.conf

    [[Gauges]]

       [[[day]]]
        [[[[outTemp]]]]
            min = -20
            max = 40
        [[[[barometer]]]]
            offset = 3
        [[[[windSpeed]]]]
            min = 0
        [[[[windDir]]]]
            arc = 2
            min = 0
            max = 360
        [[[week]]]
        [[[[outTemp]]]]
        [[[month]]]
        [[[[outTemp]]]]
        [[[year]]]
        [[[[outTemp]]]]
        tick_number = 7
        arc = 1.2
        offset = 20
        # normal or invert
        mode = normal
        color_scheme = interpolateRdBu
        invert_color_scheme = 1
        show_min_max = 1
        hide_tick_unit = 0

When i run  " weectl report run WdcReport"
i can't see gauge , just 4 whites squares , see scrennshot . 
![wdc-gauge-issues01](https://github.com/user-attachments/assets/b587d9d5-7a52-4247-8619-9254bdb3d07c)

but i display gauge  for the year ?

![wdc-gauge-issues02](https://github.com/user-attachments/assets/4a40826b-a36c-4112-8089-81e94121da5f)

could you help me please
Regards 
Remy

**Comment by Daveiano** [comment]
Okay, that's strange. Is your site publicly available so I can have a look at the "white blank gauges"?

Otherwise, it would be great if you could share your whole skin.conf file, so I could have a look, thank you!

**Comment by Ecloud44** [comment]
Hi ,
yes you can see my website on the following URL , it is a website test

https://meteo.inodes.fr/wdc/

and  my skin.conf , see attached file

Thank for your help
Regards ,Remy

**Comment by Daveiano** [comment, marked as answer]
Ok, I cant see the skin.conf file, I guess that's because you directly replied to the GitHub Notification Mail.

But I think I figured it out, your configuration is not correct, it should be like this:

```
[[Gauges]]
    tick_number = 7
    arc = 1.2
    offset = 20
    # normal or invert
    mode = normal
    color_scheme = interpolateRdBu
    invert_color_scheme = 1
    show_min_max = 1
    hide_tick_unit = 0
    [[[day]]]
      [[[[outTemp]]]]
        min = -20
        max = 40
      [[[[barometer]]]]
        offset = 3
      [[[[windSpeed]]]]
        min = 0
      [[[[windDir]]]]
        arc = 2
        min = 0
        max = 360
    [[[week]]]
    [[[[outTemp]]]]
    [[[month]]]
    [[[[outTemp]]]]
    [[[year]]]
      [[[[outTemp]]]]
```

`tick_number`, `arc` and so on must be defined for the gauges to work. In your config, these properties are only defined for the years outTemp gauge, but not for the other gauges. Properties are inheriting, so it is good practice to place default values for all gauges and override them for specific gauges when needed. For example, that's my private config: https://github.com/Daveiano/weewx-wdc-interceptor-docker/blob/main/src/skin.conf#L272-L296

I hope that solves the issue, otherwise please let me know!

**Reply by Ecloud44** [reply]
Hi , Thank a lot 
It 's work fine now !
Regards Remy
